var/spack/repos/builtin/packages/intel-mkl-dnn/package.py

```diff
@@ -13,6 +13,9 @@ class IntelMklDnn(CMakePackage):
     homepage = "https://01.org/mkl-dnn"
     url      = "https://github.com/01org/mkl-dnn/archive/v0.11.tar.gz"
 
+    version('1.0-pc2', sha256='fcc2d951f7170eade0cfdd0d8d1d58e3e7785bd326bca6555f3722f8cba71811')
```

Comment:
I wouldn't make a preview candidate the default version. Either set the previous version to `preferred=True` or drop the pc version. Also, can you update the URL? 01org redirects to intel now.

Reply:
Fix.